dbms/src/Columns/IColumn.h

```diff
@@ -193,6 +193,11 @@ class IColumn : public COWPtr<IColumn>
     /// In this special case inserting data should be zero-ending (i.e. length is 1 byte greater than real string size).
     virtual void insertDataWithTerminatingZero(const char * pos, size_t length) { insertData(pos, length); }
 
+    virtual void batchInsertDataWithTerminatingZero(size_t, const char *, size_t)
```

Comment:
why need add a virtual function in `IColumn`?

Reply:
> why need add a virtual function in `IColumn`?

This function is moved to non-virtual now